x-pack/platform/plugins/shared/dataset_quality/public/rule_types/degraded_docs/rule_form/rule_form.tsx

```diff
@@ -0,0 +1,327 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiComboBox,
+  EuiExpression,
+  EuiFormRow,
+  EuiSpacer,
+  EuiTitle,
+  EuiFormErrorText,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { DataViewSelectPopover } from '@kbn/stack-alerts-plugin/public';
+import {
+  ForLastExpression,
+  type RuleTypeParamsExpressionProps,
+} from '@kbn/triggers-actions-ui-plugin/public';
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import { DataView } from '@kbn/data-views-plugin/common';
+import { ThresholdExpression } from '@kbn/triggers-actions-ui-plugin/public';
+import { DegradedDocsRuleParams } from '@kbn/response-ops-rule-params/degraded_docs';
+import { COMPARATORS } from '@kbn/alerting-comparators';
+import { i18n } from '@kbn/i18n';
+import { isArray } from 'lodash';
+import { DataViewBase, DataViewFieldBase } from '@kbn/es-query';
+import { TimeUnitChar } from '@kbn/response-ops-rule-params/common/utils';
+import { INDEX } from '../../../../common/es_fields';
+import { useKibanaContextForPlugin } from '../../../utils';
+import { RuleConditionChart } from './rule_condition_chart';
+
+const degradedDocsLabel = i18n.translate('xpack.datasetQuality.rule.degradedDocsLabel', {
+  defaultMessage: 'degraded docs',
+});
+
+export const defaultRuleParams: Partial<DegradedDocsRuleParams> = {
+  comparator: COMPARATORS.GREATER_THAN,
+  threshold: [3],
+  timeSize: 5,
+  timeUnit: 'm',
+  groupBy: [INDEX],
```

Comment:
We are grouping by `_index` by default, but this partially true for dataStreams. DataStreams store in `_index` the backingIndexName rather than the actual indexName.
Are we ok with this @LucaWintergerst?
This is what users would see when configuring the alert
<img width="616" alt="image" src="https://github.com/user-attachments/assets/71f8501d-a9f4-43ac-a0c4-c0b364fb1b72" />
Note `_index` being not removable in the grouping.

Reply:
I think this is OK for now. If we try to hide this too much or rename it, it may cause other confusion if users look at their data in Discover